chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/SplitDeveloperJob.java

```diff
@@ -249,6 +249,14 @@ private void sendEmail(String recipientEmail, String subject, String htmlMessage
                 .sendEmail();
     }
 
+    private String getSubject(boolean success) {
+        if (success) {
+            return env.getProperty("splitDeveloper.success.emailSubject");
+        } else {
```

Comment:
Technically the else block is not needed. Just my preference to have less code when more code doesn't tell one anything but you don't need to change on my accord. The bytecode will look the same so not a performance issue...
```
        if (success) {
            return env.getProperty("splitDeveloper.success.emailSubject");
        }
        return env.getProperty("splitDeveloper.failed.emailSubject");
```

You could actually go even further and make this ternary, but that's not objectively more clear, just shorter (one liner). And doesn't offer any (pre-compiled) logic change from the current state. So I prefer the elmination of the else block. To each their own.

If this entire comment is useless, please let me know :-)